tools/StaticAnalysis/Exceptions/Az.Automation/BreakingChangeIssues.csv

```diff
@@ -0,0 +1,3 @@
+AssemblyFileName,ClassName,Target,Severity,ProblemId,Description,Remediation
```

Comment:
Please remove the breakingchangeissues.csv bc no breaking changes are introduced.

Reply:
Removed BreakingChangeIssues.csv